keras_cv/callbacks/__init__.py

```diff
@@ -11,7 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from keras_cv.callbacks.pycoco_callback import PyCOCOCallback
+try:
+    from keras_cv.callbacks.pycoco_callback import PyCOCOCallback
+except ImportError:
+    print(
```

Comment:
Probably we could use `ImportWarning` instead of a `print`.

Reply:
this sgtm -- Luke wdyt?

Reply:
lets just keep it simple for now: https://github.com/keras-team/keras-cv/blob/master/keras_cv/metrics/coco/__init__.py#L21

I'd rather fix this across the board in one PR than deviate between the import files.  Lets keep this focused to fixing master for now.